### ISSUE TITLE
I'm not sure but may I ask some code problems

### ISSUE THREAD

**Issue by MingHong1019** [psunlpgroup/CONTaiNER#2]
in line 129, 130,131, 115 of file utils.py

1. for line 115, assert a tuple will return none value and will not report error.

2. in line 129, 130,131, 115, the filtered_embedding/repeated_output_embeddings are repeted by 'len(output_embedding_mu)/filtered_instances_nos' times. So these tensors can not be reshape as (filtered_instances_nos, filtered_instances_nos), I change that to  (filtered_instances_nos, -1) and code works.

I'm not realy sure about these problems, could you please help me for that?

https://github.com/psunlpgroup/CONTaiNER/blob/main/src/utils.py#L129

**Comment by MingHong1019** [comment]
Thanks a lot for suggestion "--consider_mutual_O", and I have add that argument and it works.
By the way, the paper said that you have fine-tuning in the target domain using the label like ORG and LOCATION in Ontonotes dataset. However, I couldn't find any fine-tuning step during testing on the FewNerd dataset in the code.

**Comment by Sarathismg** [comment]
If you carefully follow the code at container.py source file, you'll notice [this line](https://github.com/psunlpgroup/CONTaiNER/blob/931728d2972cef11bc252aea9e400e0c90986fbb/src/container.py#L680). Whenever do_finetune_support_only is enabled, it will finetune the model with the support set.
